src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine ex = new Excavator();
+        Machine tr = new Truck();
+        Machine bl = new Bulldozer();
```

Comment:
Good use of abstract references instead of specific ones, which allows for polymorphism in the code.